pkg/webhook/nad/validator.go

```diff
@@ -129,6 +132,9 @@ func (v *Validator) Delete(_ *admission.Request, oldObj runtime.Object) error {
 	//do not delete nad when a subnet is using it
 	//This will also make sure nad is not deleted when VMIs,VMs are using it
 	if nadConf.Type == utils.CNITypeKubeOVN {
+		if !v.subnetEnabled {
```

Comment:
when ovn is disabled, the subnet crd is gone, related objects are gone;  the nad becomes dangling

generally, should we:
deny to create/update of ovn type nad,
allow to delete them ?